src/pretalx/orga/views/mails.py

```diff
@@ -43,7 +52,11 @@ def get_queryset(self):
         return qs
 
 
-class OutboxSend(View):
+class OutboxSend(PermissionRequired, View):
+    permission_required = 'orga.view_mails'
```

Comment:
This seems to actually send mails. So `orga.send_mails` like the `SendMail` view, which just queues mail? I am not sure whether queuing or actually sending mails or both should need `orga.send_mails`.

Reply:
Queueing should be `'edit_mails'`, while sending requires `'send_mails'`.